task/04-arrays-tasks.js

```diff
@@ -320,7 +340,10 @@ function getPositivesCount(arr) {
  *   [ 'one','one','one','zero' ]     => [ 'zero','one','one','one' ]
  */
 function sortDigitNamesByNumericOrder(arr) {
-   throw new Error('Not implemented');
+   let numbers = ['zero','one','two','three','four','five','six','seven','eight','nine'];
+   return arr.sort(function(a, b){
+      return numbers.indexOf(a) > numbers.indexOf(b);
```

Comment:
Not obvious behavior. compareFunction doesn't return a boolean value in general case.
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/sort